sdk/keyvault/Azure.Security.KeyVault.Certificates/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 
 ### Fixed
 
+- Correct ContactsPath from "/contacts/" to "/certificates/contacts/".
```

Comment:
They won't know what ContactsPath was. It's a private field. Consider the CHANGELOG to be customer-facing changes. Instead, use:
```suggestion
- Fixed issue that prevented certificate contacts from being created, enumerated, or deleted.
```